src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="logoImage"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smarphones" class="nav__link">Smartphones</a>
+          </li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link">
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#smarphones" class="nav__link">
              Smartphones
            </a>
          </li>
```
fix code style all over the page